neuralprophet/plot_forecast_plotly.py

```diff
@@ -339,6 +347,8 @@ def plot_components(
     for ax in multiplicative_axes:
         ax = set_y_as_percent(ax)
     unregister_plotly_resampler()
+    if plotly_static:
```

Comment:
@hxyue1 I am wondering, if in the 'plotly_static' case, we will display the figure twice if calling another .show() when applying the plot function?